dockerfiles/diesel-rocket-rs/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ####################################################################################################
 ## Builder
 ####################################################################################################
-FROM rust:1.62 AS builder
```

Comment:
If you update this, you should also update this one:
https://github.com/onlydustxyz/marketplace-backend/blob/main/rust-toolchain.toml